composer.json

```diff
@@ -21,7 +21,8 @@
     "require-dev": {
         "alleyinteractive/alley-coding-standards": "^1.0",
         "mantle-framework/testkit": "^0.10",
-        "nunomaduro/collision": "^5.0"
+        "nunomaduro/collision": "^5.0",
```

Comment:
Is this required? Since `@mantle-framework/testkit` already bundles it...

Reply:
https://github.com/mantle-framework/testing/blob/709076ab3e61668da048e343324c027d0737ce66/composer.json#L15

Reply:
Removed!